FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/DataFilter/IGroupMemberExtractor.cs

```diff
@@ -0,0 +1,30 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Health.Fhir.Synapse.Core.DataFilter
+{
+    public interface IGroupMemberExtractor
+    {
+        /// <summary>
+        /// Gets a list of Patients that are included in a Group, either as members or as members of nested Groups. Implicit members of a group are not included.
+        /// Patients deduplication is executed on the result.
+        /// </summary>
+        /// <param name="groupId">The group id.</param>
+        /// <param name="queryParameters">The query parameters to filter groups.</param>
+        /// <param name="groupMembershipTime">Only returns patients that were in the group at this time.</param>
+        /// <param name="cancellationToken">Cancellation token</param>
+        /// <returns>Patient id hashset.</returns>
+        public Task<HashSet<string>> GetGroupPatientsAsync(
```

Comment:
Interface fields have default `public` modifier, maybe the `public` here can be removed.

Reply:
updated.